packages/form-js-viewer/src/render/components/FormComponent.js

```diff
@@ -34,6 +34,7 @@ export default function FormComponent(props) {
       class="fjs-form"
       onSubmit={ handleSubmit }
       onReset={ handleReset }
+      noValidate
```

Comment:
Can you explain why this is needed?

Reply:
This disables browser default validation. This is something that was needed for the number field as we implement our own validation. I don't think it's crucial anymore.

Reply:
I agree. I can remember we had similar problems with the email and phone validations, so let's settle with that one and see whether it has some negative effects (which I don't think).